### DISCUSSION TITLE
Which gas sensor to use in energy dashboard?

### DISCUSSION THREAD

**Discussion by richardambridge** [BottlecapDave/HomeAssistant-OctopusEnergy#1098] [General]
Currently
https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/setup/energy_dashboard/

indicates to use;
For Gas usage you want sensor.octopus_energy_gas_{{METER_SERIAL_NUMBER}}_{{MPRN_NUMBER}}_current_accumulative_consumption
For Use an entity tracking the total costs option you want sensor.octopus_energy_gas_{{METER_SERIAL_NUMBER}}_{{MPRN_NUMBER}}_current_accumulative_cost

but i have either with kwh or m3

sensor.octopus_energy_gas_xxx_xxx_current_accumulative_consumption_kwh

sensor.octopus_energy_gas_xxx_xxx_current_accumulative_consumption_m3

What should I chose?

(Thanks for a great integration btw)

**Comment by BottlecapDave** [comment]
Hello. This guide is a little old before some entities were renamed. You'll want the kWh variation for the current gas sensors as that's what's reported natively by the home mini.
